packages/auth-magic-link-nest/src/magic-link/magic-link.strategy.ts

```diff
@@ -34,7 +33,7 @@ export class MagicLinkStrategy extends PassportStrategy(Strategy) {
                 this.config.paths.proxy ?? '',
                 this.config.paths.magicLink.validate,
               ])
-              .query({ token })
+              .query({ token, callbackUrl: user.callbackUrl })
```

Comment:
as mentioned above, we should not include the callbackUrl as a query parameter in the generated magic link if it has the "default" value:

```
...
import { DEFAULT_CALLBACK_URL } = '.../contants.ts';

.query({ 
    token,  
    ...(user.callbackUrl && user.callbackUrl !== DEFAULT_CALLBACK_URL) && { callbackUrl: user.callbackUrl }
})
```